### PR TITLE
Refactor some classes to use gravitee-common and gravitee-node

### PR DESCRIPTION
**Pull request by ytvnr**
## Issue

https://gravitee.atlassian.net/browse/APIM-3800

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-jnvrmgqhee.chromatic.com)

### CODE REVIEW

File: gravitee-apim-integration-tests/src/test/resources/proto/hello_streaming.proto [hunk 1]
```diff
@@ -1,4 +1,5 @@
-// Copyright 2015 The gRPC Authors
+//
+// Copyright © 2015 The Gravitee team (http://gravitee.io)
```
Comment (jgiovaresco) [inline gravitee-apim-integration-tests/src/test/resources/proto/hello_streaming.proto]: I don't think we should change this copyright 🙈

File: gravitee-apim-integration-tests/src/test/resources/proto/helloworld.proto [hunk 2]
```diff
@@ -1,4 +1,5 @@
-// Copyright 2015 The gRPC Authors
+//
+// Copyright © 2015 The Gravitee team (http://gravitee.io)
```
Comment (jgiovaresco) [inline gravitee-apim-integration-tests/src/test/resources/proto/helloworld.proto]: I don't think we should change this copyright 🙈

**Review by jgiovaresco — APPROVED** [review]
~I may missed it but we should remove TimeProvider class that exists in this repository?~
I confirm I miss it 🙈

File: gravitee-apim-integration-tests/src/test/resources/proto/helloworld.proto (hunk 2 above)
Comment (ytvnr) [reply gravitee-apim-integration-tests/src/test/resources/proto/helloworld.proto]: I should have not commit them but each clean install on master is modifying those proto files...

**Pull request merged**